.github/workflows/analyze-and-build.yml

```diff
@@ -0,0 +1,57 @@
+name: Analyze and Build
+
+on: push
+
+jobs:
+  js:
+    name: JavaScript Checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@master
+        with: {version: ^12.0}
+      - uses: actions/checkout@master
+      - run: yarn install --frozen-lockfile
+      - run: yarn run bundle-data
+      - run: yarn run pretty
+      - run: yarn run flow check
+      - run: yarn run lint
+      - name: Run tests
+        run: yarn run test --coverage
+      - name: Upload coverage
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+        run: bash <(curl -s https://codecov.io/bash)
+
+  bundle-android:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@master
+        with: { version: ^12.0 }
+      - uses: actions/checkout@master
+      - run: yarn install --frozen-lockfile
+      - run: yarn run bundle-data
+      - run: yarn run --silent bundle:android
+
+  bundle-ios:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@master
+        with: { version: ^12.0 }
+      - uses: actions/checkout@master
+      - run: yarn install --frozen-lockfile
+      - run: yarn run bundle-data
+      - run: yarn run --silent bundle:ios
+
+  android:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@master
+        with: { version: ^12.0 }
+      - uses: actions/setup-ruby@master
+        with: {ruby-version: ^2.6}
+      - uses: actions/checkout@master
+      - run: yarn install --frozen-lockfile
+      - run: gem install bundler && bundle check || bundle install --frozen --path vendor/bundle
+      - run: android/gradlew androidDependencies --console=plain
```

Comment:
This might need to `cd android &&` instead

Reply:
Done.